### PR TITLE
Fleet rename

### PR DESCRIPTION
**Pull request by raphaelgavache**
We transformed the updater into a local installer, with optional remote updates. Renaming to match the change and avoid confusion

### CODE REVIEW

File: cmd/updater/subcommands/bootstrap/command.go
Comment (BaptisteFoy) [inline cmd/updater/subcommands/bootstrap/command.go:1]: Should we change the path here too?

File: comp/updater/localapi/component.go
Comment (BaptisteFoy) [inline comp/updater/localapi/component.go:1]: Same

File: cmd/updater/subcommands/bootstrap/command.go
Comment (raphaelgavache) [reply cmd/updater/subcommands/bootstrap/command.go:1]: in another PR

**Review by BaptisteFoy — APPROVED** [review]

**Pull request merged**